jwst/extract_2d/grisms.py

```diff
@@ -318,10 +322,13 @@ def extract_grism_objects(input_model,
     boxes that will be used to define the 2d extraction area.
 
     For each spectral order, the configuration file contains a
-    magnitude-cutoff value. Sources with magnitudes fainter than the
-    extraction cutoff (MMAG_EXTRACT)  will not be extracted, but are
+    magnitude-cutoff value. The total list of objects to extract is limited
+    by both MMAG_EXTRACT and NBRIGHT. Sources with magnitudes fainter than the
+    extraction cutoff (MMAG_EXTRACT) will not be extracted, but are
     accounted for when computing the spectral contamination and background
-    estimates. The default extraction value is 99 right now.
+    estimates. The default value is 99 right now.
+    NBRIGHT further limits the list to the NBRIGHT brightest objects.
+    The default value is 999 right now.
```

Comment:
Should this be 300 now?

Reply:
The 300 value will be used only for crowded filed situations, which will be set in the pars-spec2pipeline ref file. The default in the code will now be 1000 (see the latest commit) for all other situations.